extensions/eclipse-che-theia-plugin-remote/src/common/plugin-uri.ts

```diff
@@ -0,0 +1,21 @@
+/**********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+
+/**
+ * Share the definition of plugin resource uris between back end and
+ * front end.
+ */
+export namespace PluginUri {
+  export const SCHEME = 'pluginresource';
```

Comment:
if it's specific to che, probably add che into the SCHEME (in case upstream provides pluginresource as well) to avoid any mismatch

Reply:
It's not really specific to che: the plugin resources at <che host>/hostedPlugin/... are also available upstream.

Reply:
it's tied to che because this protocol is not available upstream